README.md

```diff
@@ -223,5 +224,21 @@ And, in your `main.rs` or `lib.rs`:
 #![cfg_attr(feature="clippy", plugin(clippy))]
 ```
 
+## Configuration
+Some lints can be configured in a `Clippy.toml` file. It contains basic `variable = value` mapping eg.
```

Comment:
I prefer lowercase `c` so that it doesn't conflict on tab-complete with `Cargo.toml`. Also, rustfmt is lowercase too